provider/adapters/tests/data/config.json

```diff
@@ -16,6 +16,6 @@
             "path": "blob.postcard"
         }
     },
-    "fallback": "Legacy",
+    "fallback": "Hybrid",
```

Comment:
This should probably be `RuntimeManual`, as this is literally used for testing manual fallback.

Reply:
There is no diff if I make this change, so I'll make it because if anything it helps cover `RuntimeManual`.

However, note that `locales: All, fallback: Hybrid` is the way to run a pass-through datagen.

Reply:
We don't want to run pass-through datagen though. We're testing `LocaleFallbackAdapter`, which should be tested with data generated for manual fallback.